tools/msPurity/averageFragSpectra.xml

```diff
@@ -1,23 +1,14 @@
 <tool id="mspurity_averagefragspectra" name="msPurity.averageFragSpectra" version="@TOOL_VERSION@+galaxy@GALAXY_TOOL_VERSION@">
-    <description>
-        Average and filter LC-MS/MS fragmentation spectra (Inter, Intra or All)
-    </description>
-
+    <description>Average and filter LC-MS/MS fragmentation spectra (Inter, Intra or All)</description>
     <macros>
         <import>macros.xml</import>
     </macros>
-
-    <expand macro="requirements">
-    </expand>
-
- <stdio>
-        <exit_code range="1:" />
-    </stdio>
-    <command interpreter="Rscript"><![CDATA[
-        averageFragSpectra.R
-            --out_rdata="$averageFragSpectra_output_rdata"
-            --out_peaklist="$averageFragSpectra_output_tsv"
-            --pa="$pa"
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript '$__tool_directory__/averageFragSpectra.R'
+            --out_rdata=$averageFragSpectra_output_rdata
+            --out_peaklist=$averageFragSpectra_output_tsv
```

Comment:
Please quote `'` your inputs and outputs

Reply:
Thanks @lecorguille.  This is updated now - see latest changes